fivejars/openy_gated_content
# Made Shared content vitual y video as default tab.

https://github.com/ymcatwincities/openy_gated_content/issues/139

## Steps to test:
1. Pull the code from openy-139 branch.
2. Navigating to Virtual Y > Shared Content > Fetch 
2.  Once Fetch the Shared content , its should be redirect to "Virtual Y video" (Screenshot attached).

![image](https://user-images.githubusercontent.com/6428296/119674408-d0d63c80-be59-11eb-9987-702011318b35.png)
